### PR TITLE
C2 mirna yacout c4 f5 record approval of a synonym

### PR DESCRIPTION
**Pull request by Mirna-Yacout**
#81

@AmrARaouf @Omar-Hossam @karimngr @Fattouh92 @m0hamed please review

work in Synonym.rb
test: disapprove_synonym_spec.rb

### CODE REVIEW

**Comment by m0hamed** [comment]
good job. verified

**Comment by omar-mohyeldin** [comment]
@Mirna-Yacout code verified

**Comment by abulazm22** [comment]
1) Please put title
2) change id with synonym_id
3)Synonym.find(synonym_id) should return keyword and then synonym.save

**Comment by abulazm22** [comment]
Verified

File: arability/app/models/synonym.rb [hunk 1]
```diff
@@ -52,28 +52,29 @@ def self.record_suggested_synonym(synonym_name, keyword_id, approved= false)
     end
   end 
 
+  
+  # Author:
+  #  Mirna Yacout
+  # Description:
+  #  This method is to record the disapproval of the admin to a certain synonym
```
Comment (AmrARaouf) [inline arability/app/models/synonym.rb]: the body of the documentation headers have to be indented by 3 spaces not 2

File: arability/app/models/synonym.rb [hunk 2]
```diff
@@ -52,28 +52,29 @@ def self.record_suggested_synonym(synonym_name, keyword_id, approved= false)
     end
   end 
 
+  
+  # Author:
+  #  Mirna Yacout
+  # Description:
+  #  This method is to record the disapproval of the admin to a certain synonym 
+  #   in the database
```
Comment (AmrARaouf) [inline arability/app/models/synonym.rb]: if you need to extend the body to more than one line it also has to be indented by 3 spaces, not one more than the first line

**Comment by Mirna-Yacout** [comment]
@AmrARaouf done

File: arability/app/models/synonym.rb [hunk 3]
```diff
@@ -52,28 +52,28 @@ def self.record_suggested_synonym(synonym_name, keyword_id, approved= false)
...
+  #   Mirna Yacout
+  # Description:
+  #   This method is to record the disapproval of the admin to a certain synonym in the database
+  # Parameters:
+  #   id: the id of the synonym to be disapproved
+  # Success:
+  #   returns true on saving the disapproval correctly in the database
+  # Failure:
+  #   returns false if the synonym doesnot exist in the database
+  #      or if the disapproval failed to be saved in the database
```
Comment (AmrARaouf) [inline arability/app/models/synonym.rb]: same case as the other comment, should be aligned with the line preceding it not offseted by 3 more spaces

**Comment by Mirna-Yacout** [comment]
@AmrARaouf please review

**Comment by AmrARaouf** [comment]
doc and testing reviewed

**Pull request merged**